src/components/Contacts/ContactDetails/ContactDetailsHeader/ContactHeaderSection/ContactHeaderPartnerSection.tsx

```diff
@@ -32,11 +32,11 @@ export const ContactHeaderPartnerSection: React.FC<Props> = ({
     return (
       <ContactHeaderSection>
         <Typography variant="subtitle1" sx={{ fontWeight: 700 }}>
-          {t('Partner Account')}
+          {t('Partner No.')}
         </Typography>
```

Comment:
Should this be `Partner Number`? I'm thinking that it might be confusing for people translating "Partner No." to know that No is an abbreviation for number.